v2.3.0

trupovalnya opened the pull request.

trupovalnya committed:
Вычитка armors и commonevents

trupovalnya committed:
Второй батч проверки.

savannstm committed:
Merge branch 'main' into FH-Izmenene-

savannstm committed:
Комментарии

savannstm committed:
Склонения и правки

savannstm committed:
Удалить говняк

savannstm commented on translation/skills.txt:
```diff
@@ -284,13 +284,13 @@ Greater culinary skills enable you to cook meals that have\#special nutritional
+The skill to condense the effective properties of herbs and\#plants to maximize their potential. Teaches you the recipes for\#condensed blue and green herbs as well as lavender.<#>Умение выжимать концентрат из растений, чтобы улучшать их свойства.\#Даёт доступ к концентратам синих/зелёных трав и концентрату\#лаванды.
 Poison tip<#>Покрытие ядом
-Coat your weapon in poison extracted from the local plantlife.\#The poison tip takes a small toll on your sanity because of the\#constant stress of handling a poison coated weapon.<#>Покройте своё оружие ядом, добытым из местных растений.\#Покрытое ядом оружие действует на нервы, всё из-за\#стресса при обращении с подобным оружием.
+Coat your weapon in poison extracted from the local plantlife.\#The poison tip takes a small toll on your sanity because of the\#constant stress of handling a poison coated weapon.<#>Покройте своё оружие ядом, добытым из местных растений.\#Покрытое ядом оружие слегка воздействует на рассудок, всё из-за\#стресса при обращении с подобным оружием.
```
Зачем менять?

savannstm commented on translation/skills.txt:
```diff
@@ -47,14 +47,14 @@ Summon a swarm of rats to disrupt the enemy.<#>Призовите рой кры
 Snatch<#>Украсть
 Lockpicking<#>Взлом
-Blood sacrifice<#>Пожертвование кровью
+Blood sacrifice<#>Жертва кровью
```
Новый вариант звучит хуже, по моему мнению

savannstm committed:
Удаление лишних переносов строк в конце и пару изменений

savannstm commented on translation/system.txt:
```diff
@@ -29,12 +29,12 @@ To Title<#>В меню
-Agility<#>Скорость
+M.Attack<#>Маг. атака
+M.Defense<#>Маг. защита
+Agility<#>Ловкость
```
Есть ли смысл? За ловкость отвечает уклонение, агил отвечает за то, кто ходит первым, и получает второй ход

trupovalnya replied:
Неконсист был. В других случаях переведено как ловкость. Поменяю.

savannstm commented on translation/plugins.txt:
```diff
@@ -1,5 +1,5 @@
 Item: 1, Equip: 2, Synthesis: 3, kill: 4, St: 5, Options: 7, uit Game: 6<#>Предметы: 1, Экипировка: 2, Synthesis: 3, kill: 4, St: 5, Настройки: 7, uit Game: 6
-User<#>Использующий
+User<#>Использует
```
User - это сам таргет навыка

savannstm commented on translation/items.txt:
```diff
@@ -5,10 +5,10 @@
 A map of Prehevil.<#>Карта Прехевила.
 Photo album<#>Фотоальбом
-A humble little album containing all the photos you've collected\#through the years.<#>Скромный маленький альбом, содержащий все фотографии, которые\#вы собрали на протяжении многих лет.
+A humble little album containing all the photos you've collected\#through the years.<#>Скромный маленький альбом, содержащий все фотографии, которые\#вы собирали на протяжении многих лет.
```
Ну, "собирали" фактически неверно, потому что в альбоме хранятся фотографии концовок. А собрали такое абстрактное

savannstm commented on translation/items.txt:
```diff
@@ -211,18 +211,18 @@ A strong occult book written in the medieval times by Enki\#Ankarian. The chapte
 Combat manual<#>Боевое руководство
 A beginner's guide to survival and combat on a battlefield.<#>Руководство по выживанию и бою для начинающих.
 Black matter<#>Чёрная материя
-You cannot quite tell what you are looking at. You can\#recognize pasta in there. It might be edible?<#>Вы не можете точно сказать, на что вы смотрите.\#Вам это напоминает пасту. Может, оно съедобно?
```
Я думаю, вариант "вам это напоминает пасту" звучит круче

savannstm commented on translation/items.txt:
```diff
@@ -405,22 +405,22 @@ The severed head of the Kaiser.<#>Отрубленная голова Кайзе
+A condensed ball of blue herbs. Recovers great amount of\#health when digested.<#>Концентрированный шарик из синей травы.\#Восстанавливает большое количество здоровья.
 Condensed green<#>Концентрированная зелёная трава
-A condensed ball of green herbs. Offer protection against infections,\#poisoning and other mild status ailments in advance. The effect\#doesn't last long though.<#>Сконцентрированный шарик из зелёной травы. Защищает от\#инфекций, отравления и других состоянии заранее.\#К сожалению, эффект длится недолго.
+A condensed ball of green herbs. Offer protection against infections,\#poisoning and other mild status ailments in advance. The effect\#doesn't last long though.<#>Концентрированный шарик из зелёной травы. Заранее защищает от\#инфекций, отравления и других состоянии.\#К сожалению, эффект длится недолго.
```
Я думаю, "заранее" можно поменять на что-то более логичное здесь

savannstm commented on translation/items.txt:
```diff
@@ -405,22 +405,22 @@ The severed head of the Kaiser.<#>Отрубленная голова Кайзе
-A condensed ball of lavender. Recovers great amount of sanity\#when digested.<#>Сконцентрированный шарик из лаванды.\#Восстанавливает большое количество разума.
-Bench grinder<#>Настольный шлифовальный станок
-A grinding machine used to drive abrasive wheels. Commonly used\#to hand grind various tools and perform other rough grinding.\#It's very heavy and you wonder if it's worth carrying around.<#>Шлифовальный станок, используемый для полировки абразивных колес.\#Часто используется для ручной шлифовки инструментов и прочей\#шлифовальной работы. Очень тяжёлый. Стоит ли его таскать?
+A condensed ball of lavender. Recovers great amount of sanity\#when digested.<#>Концентрированный шарик из лаванды.\#Восстанавливает большое количество рассудка.
```
Рассудок, я думаю, не измеряется так

savannstm committed:
Описание для масла получше

savannstm committed:
Исправление капитализации имён богов

savannstm committed:
Замена на "встаёт наизготовку"

savannstm committed:
Обновление README

savannstm committed:
Замена "разум" на "рассудок" (кроме maps)

savannstm committed:
Начать заполнение списка изменений для версии 2.3.0

trupovalnya committed:
Третий батч вычитки.

savannstm committed:
Провер очка

savannstm committed:
Обновить скрипт для склонения

savannstm committed:
Обновить список изменений

trupovalnya committed:
Правки правок.

Заменён User, повторение слова "щупальца".

savannstm committed:
Изменить с 16-бит/канал в 8-бит

savannstm committed:
Обновить отображаемую версию

savannstm committed:
Обновить скрипт для записи

savannstm committed:
Добавить пробел в оригинальный текст "Continue"

savannstm committed:
Добавить комментарии

trupovalnya committed:
Обновление Инфо

Обновление файлов с описанием персонажей, небольшая замена теней/текста.

savannstm committed:
Касательно скрипта

savannstm committed:
Починить некорректные таргеты скиллов

savannstm committed:
Исправить нерабочий скрипт

savannstm committed:
Обновить список изменений

savannstm committed:
Вернуть Continue в предыдущий вид

savannstm committed:
Перенос вылазящих строк

savannstm committed:
Перенос вылазящих строк

savannstm committed:
Перенос вылазящих строк

trupovalnya committed:
Корректировка спрайтов

Корректировка названий навыков в соответствии с актуальным перевод, изменение угла теней для лучшей читаемости текста.

savannstm committed:
Обновить список изменений

savannstm committed:
Убрать переносы строк из GabTextов

savannstm reviewed (approved):
ПОВНИЙ АППРУВ, МЕРЖИМ В МЕЙН
